workbench/nextjs-webpack/app/api/chat/route.ts

```diff
@@ -0,0 +1,8 @@
+// THIS FILE IS JUST FOR TESTING HMR AS AN ENTRY NEEDS
```

Comment:
I dropped the symlink for the entire api directory and jsut copied the files from nextjs-turbopack to prevent symlink based errors for webpack